cfme/tests/cloud/test_tenant.py

```diff
@@ -41,3 +61,152 @@ def test_tenant_crud(tenant):
         tenant.name = fauxfactory.gen_alphanumeric(8)
     tenant.wait_for_appear()
     assert tenant.exists
+
+
+@pytest.fixture(scope="module")
+def new_tenant(appliance):
+    """This fixture creates new tenant under root tenant(My Company)"""
+    collection = appliance.collections.tenants
+
+    # Here TENANT[0] is the first tenant(parent tenant) from the tenant's list
+    tenant = collection.create(
+        name=TENANTS[0],
+        description="tenant_des{}".format(fauxfactory.gen_alphanumeric()),
+        parent=collection.get_root_tenant(),
+    )
+    yield tenant
+    tenant.delete_if_exists()
+
+
+@pytest.fixture(scope="module")
+def child_tenant(new_tenant):
+    """This fixture used to create child tenant under parent tenant - new_tenant"""
+    # Here TENANT[1] is the second tenant(child tenant) from the tenant's list
+    child_tenant = new_tenant.appliance.collections.tenants.create(
+        name=TENANTS[1],
+        description="tenant_des{}".format(fauxfactory.gen_alphanumeric()),
+        parent=new_tenant,
+    )
+    yield child_tenant
+    child_tenant.delete_if_exists()
+
+
+def check_permissions(appliance, assigned_tenant):
+    """This function is used to check user permissions for particular tenant"""
+    view = navigate_to(appliance.collections.tenants, 'All')
+    count = view.table.row_count
+    for i in range(count):
```

Comment:
I think you try like this
```
for t in view.table:
    if t["Name"].text == "test":
        t.click()
        break
```
then create Detail page view. and then assert. 

I think better solution is navigate to `Details` destination against `assigned_tenant`. I don't understand why tenant test are not using it?

Reply:
- I am agree with you on navigations.
- But I got issue related to tenant tree with respect to RBAC roles.
- I have created issue for it: https://github.com/ManageIQ/integration_tests/issues/8662
- It may change other tests related to tenant.
- So I have used this approach for this test case and added todo in this PR.